docs/sql-performance-tuning.md

```diff
@@ -228,6 +228,8 @@ The "REPARTITION_BY_RANGE" hint must have column names and a partition number is
     SELECT /*+ REPARTITION */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(c) */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(3, c) */ * FROM t
+    SELECT /*+ REPARTITION_BY_AQE */ * FROM t
```

Comment:
Other repartition hints can also be optimized by AQE, so I think this name is not precise enough.

The key point here is the user intention. To optimize for data writing, we don't need a specific number of partitions, we don't need a strict output partitioning (like partition by a column). We only need to make the output evenly distributed and be partitioned by come columns as possible as we can (best effort).

How about `REBALANCE_OUTPUT_PARTITIONS`?

Reply:
good point,  seems in previous sql config which user faced, we have no word about `output partitions`. `REBALANCE_SHUFFLE_PARTITIONS` ?

Reply:
or just `REBALANCE_OUTPUT`?

Reply:
`REBALANCE_OUTPUT` sounds good, or `REPARTITION_BY_AUTO`, `REBALANCE_PARTITION`, `REPARTITION_BY_REBALANCE`.

Reply:
I like `REBALANCE_PARTITIONS` most, as this is a partition-level thing, not row-levle.

Reply:
sounds good!

Reply:
I just saw this. What's the diff between ` /*+ REPARTITION */ ` and `/*+ REPARTITION_BY_AQE */`?